# 基板温度(L-MESのみのみ)SET
float64 BoardTemp
---

